src/Pulp/Init.purs

```diff
@@ -107,22 +133,34 @@ init initStyle force out = do
     when (dir /= cwd) (mkdirIfNotExist dir)
     writeTextFile UTF8 f.path f.content
 
-  install initStyle
+  install initStyle effOrEffect
 
   where
-    install Bower = do
+    install Bower UseEff = do
       launchBower ["install", "--save", "purescript-prelude", "purescript-console"]
       launchBower ["install", "--save-dev", "purescript-psci-support"]
 
-    install PscPackage = do
+    install Bower UseEffect = do
+      launchBower ["install", "--save", "purescript-prelude", "purescript-console", "purescript-effect"]
+      launchBower ["install", "--save-dev", "purescript-psci-support"]
+
+    install PscPackage UseEff = do
       launchPscPackage ["init"]
       launchPscPackage ["install", "eff"]
       launchPscPackage ["install", "console"]
       launchPscPackage ["install", "psci-support"]
 
+    install PscPackage UseEffect = do
+      launchPscPackage ["init"]
+      launchPscPackage ["install", "effect"]
+      launchPscPackage ["install", "console"]
+      launchPscPackage ["install", "psci-support"]
+
 action :: Action
 action = Action \args -> do
-  force      <- getFlag "force" args.commandOpts
-  pscPackage <- getFlag "pscPackage" args.globalOpts
-  out        <- getOutputter args
-  init (if pscPackage then PscPackage else Bower) force out
+  force       <- getFlag "force" args.commandOpts
+  pscPackage  <- getFlag "pscPackage" args.globalOpts
+  forceEff    <- getFlag "forceEff" args.commandOpts
```

Comment:
I think supplying both of these flags should indeed be an error, as you suggested.

Reply:
Where should this logic reside? `Init` module as well?

Reply:
Yes, I think that would make the most sense.